frontend/src/features/admin-form/create/builder-and-design/BuilderAndDesignDrawer/FieldListDrawer/field-panels/MyInfoPanel.tsx

```diff
@@ -139,7 +140,13 @@ const MyInfoText = ({
 
   if (isMyInfoDisabled) {
     return (
-      <Text>Enable MyInfo in the Settings tab to access these fields.</Text>
+      <Text>
+        Enable MyInfo in the{' '}
+        <Link as={ReactLink} to={`settings/singpass`}>
```

Comment:
Might make sense make this as a constant here
```
// frontend/src/constants/routes.ts
export const ADMINFORM_SETTINGS_SINGPASS_SUBROUTE= `${ADMINFORM_SETTINGS_SUBROUTE}/singpass`
```

Reply:
[39abc1d](https://github.com/opengovsg/FormSG/pull/6503/commits/39abc1d6845b80672b80e52f4c4cb33e49697490)

Thanks for the suggestion @KenLSM! Have also made an update to payment's settings link